ml-agents/mlagents/trainers/env_manager.py

```diff
@@ -1,18 +1,21 @@
 from abc import ABC, abstractmethod
 from typing import List, Dict, NamedTuple, Iterable
-from mlagents.trainers.brain import AllBrainInfo, BrainParameters
+from mlagents_envs.base_env import BatchedStepResult, AgentGroupSpec
+from mlagents.trainers.brain import BrainParameters
 from mlagents.trainers.policy import Policy
 from mlagents.trainers.action_info import ActionInfo
 
+AllStepResult = Dict[str, BatchedStepResult]
```

Comment:
Can you document what the keys here are? Agent IDs? Behavior names?

Reply:
They're actually AgentGroup - added this typing.